test/integration/gc_default_test.go

```diff
@@ -83,7 +83,7 @@ func TestGCDefaults(t *testing.T) {
 
 	// the /oapi endpoints should orphan by default
 	// wait for a bit and make sure that the build is still there
-	time.Sleep(2 * time.Second)
+	time.Sleep(6 * time.Second)
```

Comment:
Hi guys, Maybe using `sync.WaitGroup` you could do that without `time.Sleep`, if I'm not wrong.